<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Counter</class>
 <widget class="QWidget" name="Counter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Counter</string>
  </property>
  <widget class="QPushButton" name="button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Change period</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>View period</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
